### PR TITLE
App Portal App Id update

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
In this PR, the appId queries were taken out from requests on app portal
Convoy-app v0.0.6 was published

### CODE REVIEW

File: web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts [hunk 1]
```diff
@@ -225,7 +222,7 @@ export class ConvoyAppComponent implements OnInit {
 
 		try {
 			const eventsResponse = await this.convyAppService.request({
-				url: this.getAPIURL(`/events?groupID=${this.groupId || ''}&sort=AESC&page=${this.eventsPage || 1}&perPage=20&startDate=${startDate}&endDate=${endDate}&appId=${this.appId || ''}`),
```
Comment (horlah) [inline web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts:228]: @Oluwadaminiola Please why did you remove `perPage=20` here. Backend sending 20 by default?

Comment (Oluwadaminiola) [reply web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts:228]: yes, I confirmed with backed that they had a default page count of 20 @horlah

**Review by horlah — APPROVED** [review]
